bindings/python/src/datetime.cpp

```diff
@@ -55,6 +56,33 @@ struct time_duration_to_python
     }
 };
 
+struct time_point_to_python
+{
+    static PyObject* convert(lt::time_point tpt)
+    {
+        object result;
+        if (tpt > lt::min_time()) {
+            lt::time_point now = lt::clock_type::now();
+            time_t const tm = lt::total_seconds(tpt - now);
```

Comment:
what I suggested was to also call ``time(NULL)`` and add the number of seconds to that. However, if the version with ``system_clock`` works, I think I would prefer that